src/helpers/numberValidator.js

```diff
@@ -0,0 +1,9 @@
+const isNumber = async (num) => {
+  const isItNum = await Number.isInteger(num);
+
+  if (isItNum === true) {
+    return true;
+  }
+  return false;
+};
+export default isNumber;
```

Comment:
Simply use the isNaN JavaScript method to check if a string is a number.

Reply:
i did this. now it is working